.github/workflows/ci.yml

```diff
@@ -43,4 +43,4 @@ jobs:
       run: dotnet build --configuration Release --no-restore
 
     - name: Test ${{ matrix.version }}
-      run: dotnet test **/bin/**/${{ matrix.version }}/*Tests.dll --configuration Release --no-build --logger:"console;verbosity=detailed"
+      run: dotnet test **/bin/**/${{ matrix.version }}/*.Tests.dll --logger:"console;verbosity=detailed"
```

Comment:
This is a bug, "--configuration Release --no-build" shouldn't be there since we're testing against a DLL (and the dotnet CLI will complain about these invalid inputs).